kirin/pollers/__init__.py

```diff
@@ -0,0 +1,176 @@
+# coding=utf-8
+
+# Copyright (c) 2001, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# [matrix] channel #navitia:matrix.org (https://app.element.io/#/room/#navitia:matrix.org)
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+import logging
+import requests
+import six
+from abc import ABCMeta
+from datetime import datetime
+import kirin
+from kirin import app
+from kirin import new_relic
+from kirin.core import model
+from kirin.core.build_wrapper import wrap_build
+from kirin.utils import (
+    build_redis_etag_key,
+    get_lock,
+    make_kirin_last_call_dt_name,
+    make_kirin_lock_name,
+    manage_db_error,
+    manage_db_no_new,
+    record_input_retrieval,
+)
+from kirin.cots.model_maker import as_duration
+
+
+class Poller(six.with_metaclass(ABCMeta, object)):
```

Comment:
As mentioned by yourself, this class is actually not abstract:
```suggestion
class Poller():
```

And if it were abstract, it would be better naming it `AbstractToto`

Reply:
Done in bcfcd68dc5849d09f328fc12e392f17a612cce6b.